attack_range.conf

```diff
@@ -220,11 +223,15 @@ caldera_password = I-l1ke-Attack-Range!
 [environment]
 # specify your Attack Range environment by enabling (1) or disabling (0) machines
 
+splunk_server = 1
```

Comment:
Not sure if we want optional splunk_server

Reply:
Cool, I've taken out the optional bits for the Splunk server so that it always gets built.